spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/TraceFilter.java

```diff
@@ -80,9 +80,6 @@
 	protected static final String TRACE_ERROR_HANDLED_REQUEST_ATTR = TraceFilter.class.getName()
 			+ ".ERROR_HANDLED";
 
-	public static final String DEFAULT_SKIP_PATTERN =
```

Comment:
I'd leave this field, deprecate it and point it to the `SleuthWebProperties .DEFAULT_SKIP_PATTERN`. If anyone is using this field it will still be applicable